File: socket/msg_proxy_server/src/main/java/cn/facenom/models/Message.java
package cn.facenom.models;

import lombok.Data;

/**
 * @author zyh
 * @version 1.0
 * @since 2023-03-05
 */
@Data
public class Message {

    private String address;

    private String port;

    private String host;

    private String content;
}

File: socket/msg_proxy_client/lib/models/message.dart
class Message {
  
  final String address;
  
  final String port;

  final String hostName;
  
  final String content;
  
  final bool isOwner;
  
  Message({
    required this.address, 
    required this.port,
    required this.hostName,
    required this.isOwner,
    required this.content,
  });
  
}